addon/mixins/unauthenticated-route-mixin.js

```diff
@@ -34,6 +35,12 @@ export default Ember.Mixin.create({
   */
   session: service('session'),
 
+  _isFastBoot: computed(function() {
+    const fastboot = getOwner(this).lookup('service:fastboot');
+
+    return fastboot ? fastboot.get('isFastBoot') : false;
+  }),
```

Comment:
same code as in `authenticated-route-mixin.js` -- Potentially worth creating either a base mixin, or a util containing things like this?